agent/engine/serviceconnect/manager_linux_test.go

```diff
@@ -150,6 +150,28 @@ func TestGetECSAgentLogPathContainer(t *testing.T) {
 			}
 			actualPath := getECSAgentLogPathContainer()
 			assert.Equal(t, tc.expected, actualPath)
+
+		})
+	}
+}
+
+func TestGetSupportedAppnetInterfaceVerToCapabilities(t *testing.T) {
```

Comment:
Could we add negative tests cases to verify how errors will be handled : )

Reply:
GetSupportedAppnetInterfaceVerToCapabilities() returns errors only for unsupported platforms so adding a test case that handles a empty string for appNetAgentVersion.